models/infra_schema.js

```diff
@@ -0,0 +1,13 @@
+const { connector } = require("./databaseUtil");
+
+const infraStructureSchema = {
+    infraID: {type: Number, required: true},
+    infraName: {type: String, required: true},
+    infraType: {type: String, required: true},
+    infraWing: {type: String, required: true},
+    floor: {type: Number, required: true},
+    capacity: {type:Number, required: true}
+};
+
+const infraStructure = new connector.model("Infra strucutre", infraStructureSchema);
```

Comment:
please fix this line
```suggestion
const infrastructureModel = new connector.model("Infrastructure", infrastructureSchema);
```